File: WebApplication/src/modules/candidate/models/candidate-applications.dto.ts
import { ApplicationSearchResultItemDto } from 'src/modules/application/models/application-search-result-item.dto';
import { CandidateCommitDto } from './candidate-commit.dto';

export class CandidateApplicationsDto {
	candidateCommit: CandidateCommitDto;
	applications: ApplicationSearchResultItemDto[];
	hasOtherCommits: boolean;
}

File: VisaD.Application/Candidates/Dtos/CandidateApplicationsDto.cs
using System.Collections.Generic;
using VisaD.Application.Applications.Dtos;

namespace VisaD.Application.Candidates.Dtos
{
    public class CandidateApplicationsDto
    {
        public CandidateCommitDto CandidateCommit { get; set; }
        public IEnumerable<ApplicationSearchResultItemDto> Applications { get; set; }
        public bool HasOtherCommits { get; set; }
    }
}
